source/com.microsoft.tfs.util/src/com/microsoft/tfs/util/messages.properties

```diff
@@ -1,4 +1,24 @@
 AdvisoryFileLock.InterruptedAcquiringLockFormat=Interrupted while acquiring blocking lock on FileChannel for file {0}
+EmailAddressParser.AddressTooLong=The e-mail address cannot be longer that 256 characters.
+EmailAddressParser.DomainCannotEndWithDot=The domain part cannot end with the dot (''.'') character
+EmailAddressParser.DomainCannotStartWithDot=The domain part cannot start with the dot (''.'') character
+EmailAddressParser.DomainTooLong=The domain part cannot be longer than 253 characters.
+EmailAddressParser.DotAtBeginning=The dot (''.'') character is not allowed at the beginning of the local part
+EmailAddressParser.DotAtEnd=The dot (''.'') character is not allowed at the end of the local part
+EmailAddressParser.EmptyLocalpart=The local part cannot be empty.
+EmailAddressParser.LocalPartTooLong=The local part cannot be longer that 64 characters.
+EmailAddressParser.ManyDots=Multiple dot (''.'') characters are not allowed
+EmailAddressParser.MissingDomainpart=Missing domain part.
+EmailAddressParser.SubdomainCannotBeEmpty=The subdomain name cannot be empty.
+EmailAddressParser.SubdomainCannotEndWithHyphen=The subdomain name cannot end with the hyphen (''-'') character.
+EmailAddressParser.SubdomainCannotStartWithHyphen=The subdomain name cannot start with the hyphen (''-'') character.
+EmailAddressParser.SubdomainTooLong=The subdomain name cannot be longer than 63 characters.
+EmailAddressParser.TooManyDomainLevels=The domain part cannot more that 127 levels.
+EmailAddressParser.UnallowedCharacterFormat=Unallowed character ''{0}'' at position {1}.
+EmailAddressParser.UnexpectedEol=Unexpected end of string in the e-mail address.
+EmailAddressParser.UnexpectedEolInQuoteFormat=Unexpected end of string in the e-mail address. Probably the last quote mark at position {0} was not closed.
+EmailAddressParser.UnquotedAtInQuoteFormat=Unexpected unquoted ''@'' character inside of quoted substring at position {0}. Probably the last quote mark at position {1} was not closed.
+EmailAddressParser.WrongCaracterInSubdomain=The subdomain name can cosist only of alphanumeric characters ang hyphens (''-'').
```

Comment:
Typographical error in `Caracter` (constant/property name), as well as message: `cosist` and `ang`.